lib/elixir/lib/list.ex

```diff
@@ -88,10 +88,39 @@ defmodule List do
       iex> 'abc'
       'abc'
 
+  Even though the representation changed the raw data does remain a list of
+  numbers, which can be handled as such:
+
+      iex> inspect('abc', charlists: :as_list)
+      "[97, 98, 99]"
+      iex> Enum.map('abc', fn num -> 1000 + num end)
+      [1097, 1098, 1099]
+
+  You can use the `IEx.Helpers.i/1` helper to get a condensed rundown on
+  charlists in iex when you encounter them:
+
+      > i 'abc'
```

Comment:
Not sure what to do here. We can't use `iex>` because the doctest won't pass but this format may be confusing. Maybe we should just say in the docs something like "call `i 'abc'`" and remove the snippet from here.

Reply:
In addition, the output might change in the future

Reply:
Yeah it's a bit of a bummer. I feel like the output is very clear in what's going on, while at the same time giving people the awareness of "I can do that whenever I encounter those things in the future".